src/common/interfaces/list.interface.ts

```diff
@@ -6,3 +6,12 @@ export interface List<T> {
     after?: string;
   };
 }
+
+export interface DeserializedList<T> {
```

Comment:
Once this body of work is done I'm going to rename `DeserializedList` to `List` and then I'll rename `List` to `ListResponse` to match the rest of the naming conventions.